app/src/main/java/org/mozilla/fenix/settings/advanced/LocaleViewHolders.kt

```diff
@@ -41,7 +53,12 @@ class SystemLocaleViewHolder(
     override fun bind(locale: Locale) {
         locale_title_text.text = itemView.context.getString(R.string.default_locale_text)
         // Use the device locale for the system locale subtitle.
-        locale_subtitle_text.text = locale.getDisplayName(locale).capitalize(locale)
+        if (locale.language == "zh" && locale.country == "TW") {
```

Comment:
Do we also need to handle `locale.country == "CN"` here?

Reply:
I didn't handle `locale.country == "CN"` because many manufacturers in the mainland of China also provide traditional Chinese in their phone settings. If I make the display name here "Simplified Chinese" with all `locale.language == "zh" && locale.country == "CN"`, it might be weird for those who set their phones in traditional Chinese while their location is "CN".

Reply:
update: Using [locale.script](https://developer.android.com/reference/java/util/Locale#getScript()) to handle SystemLocaleViewHolder